File: app/src/main/java/com/devdream/cookall/core/dto/ChatUserDTO.java
package com.devdream.cookall.core.dto;

import com.stfalcon.chatkit.commons.models.IUser;

public class ChatUserDTO implements IUser {

    public String id;
    public String name;
    public String avatar;

    @Override
    public String getId() {
        return id;
    }

    @Override
    public String getName() {
        return name;
    }

    @Override
    public String getAvatar() {
        return avatar;
    }

}

File: app/src/main/java/com/devdream/cookall/core/dto/ChatUserDTO.kt
package com.devdream.cookall.core.dto

import com.stfalcon.chatkit.commons.models.IUser

class ChatUserDTO : IUser {

    var id: String? = null
    var name: String? = null
    var avatar: String? = null

    override fun getId(): String? {
        return id
    }

    override fun getName(): String? {
        return name
    }

    override fun getAvatar(): String? {
        return avatar
    }

}
